### ISSUE TITLE
Submission possible after deadline

### ISSUE THREAD

**Issue by FatzkeFTW** [esamarathon/user-management#76]
It's currently possible to submit after the deadline by (I think) keeping the tab where you're filling out the form open until after submissions are closed. Hitting "save" then only records the game name though.

**Comment by CBenni** [comment]
Do you know if this is still an issue? I think I've fixed this a long time ago, in a previous patch.

**Comment by FatzkeFTW** [comment]
Yes, it's still an issue. I just tried it using a test event. Everything is being recorded as well, not just the game name.
